README.md

```diff
@@ -5,7 +5,7 @@
 [![Codecov][codecov-src]][codecov-href]
 [![License][license-src]][license-href]
 
-Smart Configuration Loader.
+c12 (pronounced as /siːtwelv/, like c-twelve) is a smart configuration loader.
```

Comment:
Should I keep the phrase "Smart Configuration Loader" as quote and write pronunciation separately, like below?

---

> Smart Configuration Loader

## Pronunciation

c12 is pronounced as /siːtwelv/, like c-twelve.

## Features

...

Reply:
Seems good like this too but i generally prefer to split to two lines

Reply:
Sorry, but do you mean which is preferable for you?
The code above or the markdown content?
Or, anything else?

Reply:
Well, do you mean like this?

```suggestion
Smart Configuration Loader.

c12 is pronounced as /siːtwelv/, like c-twelve.
```

Reply:
Let's keep it like now 👍🏼